src/services/signedTransaction.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import { Request, Response } from "express";
+
+import { assertNever, contentTypeHeaders, errMsgs, graphqlEndpoint, UtilEither} from "../utils";
+
+const submissionEndpoint = 'https://adalite.io/api/txs/submit';
+
+interface SubmissionData {
+    txHash: string;
+    txBody: string;
+
+};
+
+export const handleSignedTx = async (req: Request, res: Response) => { 
+  if(!req.body.signedTx)
+      throw new Error ("No signedTx in body");
+  const submission : SubmissionData = { txHash: req.body.signedTx
```

Comment:
Both of these are wrong.

1) It looks like the Adalite backend wants the `txBody` as a hex string (whereas we use base64 for `signedTx`). You will need to convert the string from base64 to base16

2) To get the txHash from a base64 transaction, you will have to use the Cardano WebAssembly bindings. Here is code I wrote that turns a hex-encoded transaction to a transaction id https://repl.it/repls/TartUntrueAutoresponder

Reply:
:+1: